src/foundry/client/pixi/webgl/shaders/lighting/effects/star-light.d.mts

```diff
@@ -0,0 +1,15 @@
+export {};
+
+declare global {
+  /**
+   * A disco like star light.
+   */
+  class StarLightColorationShader extends AdaptiveColorationShader {
+    /**
+     * @defaultValue `true`
+     */
+    static override forceDefaultColor: boolean;
+
+    static override fragmentShader: string | ((...args: any[]) => string);
```

Comment:
You'll notice that after this point I stop mentioning `AnyFunction` and the `Uniforms` in each class's namespace. I just didn't want to drown you in even more review comments about the same thing but it still applies